src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4424,14 +4457,26 @@ string buildBindBack()
 					}
 				}
 
-				var contextFunction = XBindExpressionParser.Rewrite("___tctx", rawFunction, dataTypeSymbol, _metadataHelper.Compilation.GlobalNamespace, isRValue: true, FindType);
-				return $".BindingApply({applyBindingParameters} => /*defaultBindMode{GetDefaultBindMode()}*/ global::Uno.UI.Xaml.BindingHelper.SetBindingXBindProvider(___b, null, ___ctx => ___ctx is {GetType(dataType)} ___tctx ? ({contextFunction}) : (false, default), {buildBindBack()} {pathsArray}))";
+				var contextFunction = XBindExpressionParser.Rewrite("___tctx", rawFunction, dataTypeSymbol, _metadataHelper.Compilation.GlobalNamespace, isRValue: true, _xBindCounter, FindType);
+				if (_fileDefinition.FilePath.Contains("When_xLoad_DataTemplate_In_ResDict_Global", StringComparison.Ordinal))
+				{
+					//Debugger.Launch();
+				}
```

Comment:
Remove

Reply:
@jeromelaban Already removed in the last commit